akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/BehaviorSetup.scala

```diff
@@ -66,11 +72,16 @@ private[akka] final class BehaviorSetup[C, E, S](
   def selfClassic: ClassicActorRef = context.self.toClassic
 
   private var mdcPhase = PersistenceMdc.Initializing
+  
+  private val internalLogger = 
+    LoggerFactory.getLogger(context.log.getName + "Internal") 
```

Comment:
Just to double check: This isn't racy with the logic to guess/sett custom names in the logger because that would already have been done in the user setup block of the actor? (see `akka.persistence.typed.scaladsl.LoggerSourceSpec`)

Reply:
Since `context.log.getName` can change over time via the `setLoggerName` I think this should be a `def`.